README.md

```diff
@@ -1,73 +1,83 @@
 [![Telegram][telegram-badge]][telegram-url]
-[![Slack][slack-badge]][slack-url]
-[![Gitter][gitter-badge]][gitter-url]
-[![Google Groups][groups-badge]][groups-url]
+[![GitHub Discussions][discussions-badge]][discussions-url]
+[![Stack Overflow][stackoverflow-badge]][stackoverflow-url]
 
-https://tarantool.io/en/
+[Tarantool][tarantool-url] is an in-memory database and application server.
 
-Patch submissions and discussion of particular patches https://lists.tarantool.org/mailman/listinfo/tarantool-patches/
-
-General development discussions https://lists.tarantool.org/mailman/listinfo/tarantool-discussions/
-
-Tarantool is an in-memory database and application server.
+It is distributed under [BSD 2-Clause][license] terms.
 
 Key features of the application server:
- * 100% compatible drop-in replacement for Lua 5.1,
-   based on LuaJIT 2.1.
-   Simply use `#!/usr/bin/tarantool` instead of
-   `#!/usr/bin/lua` in your script.
- * full support for Lua modules and a rich set of
-   own modules, including cooperative multitasking,
-   non-blocking I/O, access to external databases, etc
+
+* 100% compatible drop-in replacement for Lua 5.1, based on LuaJIT 2.1.
+  Simply use `#!/usr/bin/tarantool` instead of `#!/usr/bin/lua` in your script.
+* Full support for Lua modules and a rich set of [own modules][modules],
+  including cooperative multitasking, non-blocking I/O, access to external
+  databases, [persistent queues][queue], [sharding][vshard], [cluster and
+  application management framework][cartridge], etc.
 
 Key features of the database:
- * ANSI SQL, including views, joins, referential
-   and check constraints
- * MsgPack data format and MsgPack based
-   client-server protocol
- * two data engines: 100% in-memory with
-   optional persistence and an own implementation of LSM-tree, 
-   to use with large data sets
- * multiple index types: HASH, TREE, RTREE, BITSET
- * asynchronous master-master replication
- * authentication and access control
- * the database is just a C extension to the
-   application server and can be turned off
-
-Supported platforms are Linux/x86, FreeBSD/x86 and OpenBSD/x86, Mac OS X.
-
-Tarantool is ideal for data-enriched components of
-scalable Web architecture: queue servers, caches,
-stateful Web applications.
-
-To download and install Tarantool as a binary package for your OS, please visit
-https://tarantool.io/en/download/.
-
-To build Tarantool from source, see detailed instructions in the Tarantool
-documentation at https://tarantool.io/en/doc/2.1/dev_guide/building_from_source/.
-
-Please report bugs at https://github.com/tarantool/tarantool/issues
-We also warmly welcome your feedback in the discussion mailing
-list, tarantool@googlegroups.com.
+
+* MessagePack data format and MessagePack based client-server protocol.
+* Two data engines: 100% in-memory with optional persistence and an own
+  implementation of LSM-tree, to use with large data sets.
+* Multiple index types: HASH, TREE, RTREE, BITSET.
```

Comment:
AFAIK, `HASH` is deprecated. Perhaps we should not mention it in new docs. @Mons can tell more.

Reply:
AFAIU these all just talks at the moment (in fact, these talks going two years or so). There is no final decision. We have no deprecation warning or statements in the documentation — nothing official.

I asked @Mons and he said 'Don't know. Here we can leave it.'

Reply:
So I'll leave it.